apps/frontend/src/app/Components/Artifact/RVSlide.tsx

```diff
@@ -45,28 +58,58 @@ export default function RVSlide({
         overflow: 'visible',
       }}
     >
-      <CustomNumberInput
-        value={sliderLow}
-        onChange={(val) => setLow(clamp(val ?? 0, 0, levelHigh))}
-        sx={{ px: 1, pl: 2, width: 100, borderRadius: '4px 0 0 4px' }}
-        inputProps={{ sx: { textAlign: 'right' } }}
-        startAdornment={'RV: '}
-        disabled={disabled}
-      />
+      <Box
+        sx={{ width: 100, height: 30, display: 'flex', alignItems: 'center' }}
+      >
+        <Button
```

Comment:
This behaviour seems kind of complicated, I wonder if it makes sense to just have both RV filters, and remove this `useMaxRV` complexity?